fhir-config/src/main/java/com/ibm/fhir/config/FHIRRequestContext.java

```diff
@@ -94,6 +95,7 @@ public void setTenantId(String tenantId) throws FHIRException {
      * @param base64
      * @throws FHIRException if the given value is not a valid Base64 string
      */
+    @Deprecated
```

Comment:
maybe better to show the reason like it's not needed anymore or not useful anymore because it's handled in fhirdaoimp..

Reply:
I agree and so I just left a suggestion that I believe would address the comment.  Should we add a similar javadoc on the `getTenantKey()` method as well?

Reply:
Picked up the suggestion and applied... I'm going to apply to getTenantKey as well

Reply:
Applied in next commit